### DISCUSSION TITLE
Configuration of xandikos

### DISCUSSION THREAD

**Discussion by lamyergeier** [jelmer/xandikos#167] [Q&A] [answered]
I am able to access xandikos using http://localhost:8080/user, my IP address is 192.168.148.22 . But I am unable to access it using http://192.168.148.22:8080/user. This is necessary for DAVx5 on Android. How to make it happen? I did this in Radicale by changing the config file.

Also, Radicale creates a file called as `.Radicale.props` to specify the color, name and other details about the calendar. Is `.xandikos` an equivalent file? If yes what its syntax (what keys to use)?

**Comment by jelmer** [comment, marked as answer]
You can specify the address for xandikos to listen on using the --listen-address flag, e.g. "./bin/xandikos --listen-address 0.0.0.0"

.xandikos is for collection-specific metadata (which you'd probably be changing through your CalDAV/CardDAV client), rather than configuration for Xandikos.
